doc/select.md

```diff
@@ -0,0 +1,204 @@
+# Select examples
+
+## Filtering
+
+The second parameter passed to ``crud.select`` is an array of conditions. Below are examples of filtering data using these conditions. 
```

Comment:
Maybe smth like 

CRUD allows to filter tuples by conditions.
Each condition can use field name/number or index name. The first condition that uses index name is used to iterate over space (if there is no conditions that match index names, full scan is performed). Other conditions are used as additional filters.